src/main/java/io/cdap/plugin/servicenow/source/ServiceNowMultiSource.java

```diff
@@ -75,7 +75,14 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
     // Since we have validated all the properties, throw an exception if there are any errors in the collector.
     // This is to avoid adding same validation errors again in getSchema method calls
     collector.getOrThrowException();
-    stageConfigurer.setOutputSchema(null);
```

Comment:
We can keep the same behavior as other multi source plugins . ex - https://github.com/data-integrations/multi-table-plugins/blob/develop/src/main/java/io/cdap/plugin/MultiTableDBSource.java#L84
Revert the change in this file.

Reply:
Reverted 5b4f0c9cb9d3fb2d99126c231f4f9283f566e69a